ee/tables/jwt/table.go

```diff
@@ -0,0 +1,177 @@
+package kolide_jwt
```

Comment:
package names should match directories. `jwt` is a reasonable package name here. The collision with the other jwt library isn't going to be an issue in practice. (And it's pretty common in go)

Reply:
Done